x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/ElasticInferenceServiceSparseEmbeddingsRequestManager.java

```diff
@@ -72,7 +73,8 @@ public void execute(
             truncator,
             truncatedInput,
             model,
-            traceContext
+            traceContext,
+            XPackPlugin.getSharedLicenseState().getOperationMode()
```

Comment:
Initially I wanted to put this operation into the `*Request` classes, but I moved them here for two reasons:
* `XPackPlugin.getSharedLicenseState()` is not populated in unit tests and I couldn't find a way to set it.
* The request class should only be responsible for constructing a request from passed input, and not for fetching configuration.

Reply:
Update: some other tests are failing now for the same reason (`XPackPlugin.getSharedLicenseState()` is null in those tests). Looking for a better solution...